src/main/java/com/nerodesk/om/mock/MkFriends.java

```diff
@@ -40,9 +41,8 @@
  * @author Yegor Bugayenko (yegor@teamed.io)
  * @version $Id$
  * @since 0.3
- * @todo #103:30min/DEV This class is not implemented and doesn't work. Let's
- *  implement it and test in a unit test. Let's use some primitive mechanism,
- *  based on file names.
+ * @todo #103:30min/DEV This class does not have any tests implemented yet.
+ *  Let's implement tests for all existent methods (leader, names, add, eject).
  */
```

Comment:
@pinaf I would suggest add `@EqualsAndHashCode` because `AwsFriends` has it. Maybe `ToString` could be usefull too.